sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -154,7 +162,59 @@ func App() *cli.App {
 	return app
 }
 
-func runCtr(containerdSocket string, namespace string, containerID string, source string, superpowered bool) error {
+// Used to define valid container types
+type ContainerType string
+
+const (
+	Host      ContainerType = "host"
+	Bootstrap ContainerType = "bootstrap"
+)
```

Comment:
GitHub lost my comment.

Please update this to keep the types private.

```suggestion
// Used to define valid container types
type containerType string

const (
	containerTypeHost      containerType = "host"
	containerTypeBootstrap containerType = "bootstrap"
)
```